src/main/java/ch/njol/skript/lang/function/DynamicFunctionReference.java

```diff
@@ -162,7 +163,7 @@ public String toString() {
 		this.checkedInputs.put(input, null); // failure case
 		if (signature == null)
 			return null;
-		boolean varArgs = signature.getMaxParameters() == 1 && !signature.getParameter(0).single;
+		boolean varArgs = signature.getMaxParameters() == 1 && !signature.parameters().firstEntry().getValue().single();
 		Expression<?>[] parameters = input.parameters();
```

Comment:
can you rename this to be more clear about when the input params are being used vs when the signature params are used?